### PR TITLE
[include] fix `#include` style for public OT headers

### PR DESCRIPTION
**Pull request by abtink**
This commit updates `#include` of public OT headers to follow the OT style guide and use brace (`<`) and (`>`) style.

### CODE REVIEW

File: include/openthread/instance.h [hunk 1]
```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (441)
+#define OPENTHREAD_API_VERSION (442)
```
Comment (jwhui) [inline include/openthread/instance.h:56]: Please rebase and increment :)

Comment (abtink) [reply include/openthread/instance.h]: Done!

File: include/openthread/instance.h [hunk 2]
```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (443)
+#define OPENTHREAD_API_VERSION (444)
```
Comment (jwhui) [inline include/openthread/instance.h:56]: Need to increment again :)

Comment (abtink) [reply include/openthread/instance.h]: Rebased and incremented.

**Review by jwhui — APPROVED** [review]

**Pull request merged**